codacy/values-production.yaml

```diff
@@ -9,8 +9,9 @@ global:
     #   privateKey: ""
 
   githubEnterprise: {}
-    # If you set this provider configuration, you must save it on the admin/integrations UI.
     # enabled: "false"
+    # clientId: ""
+    # clientSecret: ""
```

Comment:
I suggest placing these two configurations just before the `app.*` ones so they're all grouped together.